processor/batchprocessor/splitlogs_test.go

```diff
@@ -104,8 +119,51 @@ func TestSplitLogsMultipleResourceLogs_split_size_greater_than_log_size(t *testi
 	assert.Equal(t, splitSize, split.LogRecordCount())
 	assert.Equal(t, 40-splitSize, td.LogRecordCount())
 	assert.Equal(t, 1, td.ResourceLogs().Len())
-	assert.Equal(t, "test-log-int-1-19", split.ResourceLogs().At(0).InstrumentationLibraryLogs().At(0).Logs().At(0).Name())
-	assert.Equal(t, "test-log-int-1-0", split.ResourceLogs().At(0).InstrumentationLibraryLogs().At(0).Logs().At(19).Name())
-	assert.Equal(t, "test-log-int-0-19", split.ResourceLogs().At(1).InstrumentationLibraryLogs().At(0).Logs().At(0).Name())
-	assert.Equal(t, "test-log-int-0-15", split.ResourceLogs().At(1).InstrumentationLibraryLogs().At(0).Logs().At(4).Name())
+	assert.Equal(t, "test-log-int-0-0", split.ResourceLogs().At(0).InstrumentationLibraryLogs().At(0).Logs().At(0).Name())
+	assert.Equal(t, "test-log-int-0-19", split.ResourceLogs().At(0).InstrumentationLibraryLogs().At(0).Logs().At(19).Name())
+	assert.Equal(t, "test-log-int-1-0", split.ResourceLogs().At(1).InstrumentationLibraryLogs().At(0).Logs().At(0).Name())
+	assert.Equal(t, "test-log-int-1-4", split.ResourceLogs().At(1).InstrumentationLibraryLogs().At(0).Logs().At(4).Name())
+}
+
+func BenchmarkSplitLogs(b *testing.B) {
```

Comment:
Could you record the benchmark results before and after the main change?

Reply:
But good catch

Reply:
Done